backends/bmv2/bmv2.cpp

```diff
@@ -104,9 +104,13 @@ int main(int argc, char *const argv[]) {
     // Generate a PI control plane API for this program if requested.
     if (!options.p4RuntimeFile.isNullOrEmpty()) {
         std::ostream* out = openFile(options.p4RuntimeFile, false);
+        std::ostream *outEntries = nullptr;
+        if (!options.p4RuntimeEntriesFile.isNullOrEmpty())
+            outEntries = openFile(options.p4RuntimeEntriesFile, false);
```

Comment:
Please add a warning if a runtime entries file is specified on the command line without the --p4runtime-file.

Reply:
Will do